packages/@okta/vuepress-site/docs/reference/api/captchas/index.md

```diff
@@ -0,0 +1,922 @@
+---
+title: CAPTCHAs
+category: management
+---
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" />
+
+> **Note:** This feature is only available as a part of the Okta Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
+
+# CAPTCHAs API
+
+The Okta CAPTCHAs API provides operations to manage CAPTCHAs and org-wide CAPTCHA settings.
+
+## Get started
+
+Explore the CAPTCHAs API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/c51413d80cc8e88fd101?action=collection%2Fimport)
+
+## CAPTCHA operations
+
+The CAPTCHAs API has the following CRUD operations:
+
+* [Create CAPTCHA](#create-captcha)
+* [Get CAPTCHA](#get-captcha)
+* [Get all CAPTCHAs](#get-all-captchas)
+* [Update CAPTCHA](#update-captcha)
+* [Delete CAPTCHA](#delete-captcha)
+
+### Create CAPTCHA
+
+<ApiOperation method="post" url="/api/v1/captchas" />
+
+Creates a CAPTCHA object
+
+> **Note:** One organization can only have one CAPTCHA.
```

Comment:
Suggest: Note: One organization can only have one CAPTCHA provider configured at a time (or _CAPTCHA type configured..._ to align with the API property)